dmodel/dm-query.c

```diff
@@ -1324,6 +1350,7 @@ dm_query_to_string (DmQuery *self)
   DUMP_ENUM(order, DM_TYPE_QUERY_ORDER, DM_QUERY_ORDER_ASCENDING)
   DUMP_UINT(limit, G_MAXUINT)
   DUMP_UINT(offset, 0)
+  DUMP_UINT(cutoff, -1)
```

Comment:
Kind of a nitpick since this function is for debugging, but I think this is going to say something odd (such as `#cutoff: 4294967295` if cutoff == -1 ;) Need a signed integer version of that macro.

Reply:
I'm gonna make a more generic version of that macro so it's more correct.

But FWIW, it's not gonna print anything if cutoff == -1 because it's the default.

Reply:
Oh yeah. I was wondering why it had a default parameter and completely glazed over the part where it checks. Cool beans :)